packages/cli/src/util/runner.ts

```diff
@@ -0,0 +1,38 @@
+import { getHttpOperationsFromResource } from '@stoplight/prism-http';
+import { IPrismHttpServer } from '@stoplight/prism-http-server/src/types';
+import * as chokidar from 'chokidar';
+import { CreatePrismOptions } from './createServer';
+
+export function runPrism(createPrism: Function, options: CreatePrismOptions, spec: string) {
+  return createPrism(options).then((server: IPrismHttpServer) => {
+    if (server) {
```

Comment:
this check is needed because `server` is only returned by the fork https://github.com/stoplightio/prism/blob/5c7a6958275f9b838593821654a9348834172223/packages/cli/src/util/createServer.ts#L29